src/benchmarks/micro/runtime/JetStream/Compat.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace JetStream {
+    public static class Compat {
+        public static readonly Random Random = new Random ();
+
+        public static double random () {
+            return Random.NextDouble ();
+        }
+
+        public static void push<T> (this List<T> list, T item) {
```

Comment:
nit: this is not used now and can be removed

```suggestion
        public static void push<T> (this List<T> list, T item) {
```